ONSdigital/dp-frontend-search-controller
# Fix/invalid params cause oom

### What

✅ [Fix invalid page parameters causing app OOM](https://jira.ons.gov.uk/browse/DIS-1999)
- Refactor ordering of api calls and where concurrent calls happen
- Added missing page mapping for validation errors
- Added missing toggle from template to handle missing search data

### How to review

Sense check
CI checks pass

![image](https://github.com/user-attachments/assets/50513399-7471-4e39-a0fb-b24ddfd96025)

Check locally
- Port forward to sandbox api router `dp ssh sandbox web 1 -p 23200:localhost:10800`
- Run `dp-design-system` `nvm use && npm run dev`
- Pull this branch and run controller `make debug`
- See the error for yourself at: http://localhost:25000/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/bulletins/uklabourmarket/previousreleases?page=1000 

### Who can review

!me
